.github/workflows/bump-dependencies.yml

```diff
@@ -137,7 +137,14 @@ jobs:
           fi
           if [[ $existing_pr ]]; then
             existing_title="$(gh pr view "${existing_pr}" --json title --jq .title)"
-            gh pr edit "${existing_pr}" --title "${pr_title}" --body "${body}"
+            # Use Github's API directly instead of using `gh edit`.
+            # The latter internally queries all fields of the PR which leads to
+            # a permission error as the workflow does not have access to
+            # organization projects:
+            gh api --silent -H "Accept: application/vnd.github+json" --method PATCH \
```

Comment:
Is `--silent` a good idea?  I know the script will fail if `gh` returns non-zero - but will the log contain the error message with `--silent`?

Reply:
> but will the log contain the error message with --silent?

I think so.
Without `--silent`, the output will always contain the full JSON which sounds rather annoying.

Reply:
If we want stderr but not stdout... seems to me we should silence stdout (`>&-`) rather than both.

Reply:
I've just run some tests:

**GET existing pull request without --silent**
```
$ gh api  -H "Accept: application/vnd.github+json" --method GET "/repos/jamulussoftware/jamulus/pulls/2801"
{
  "url": "https://api.github.com/repos/jamulussoftware/jamulus/pulls/2801",
  "id": 1033348737,
```

**GET existing pull request with --silent**
```
$ gh api --silent -H "Accept: application/vnd.github+json" --method GET "/repos/jamulussoftware/jamulus/pulls/2801"
$
```

**GET non-existing pull request with --silent**
```
$ gh api --silent -H "Accept: application/vnd.github+json" --method GET "/repos/jamulussoftware/jamulus/pulls/0"
gh: Not Found (HTTP 404)
```

**GET non-existing pull request without --silent**
```
$ gh api -H "Accept: application/vnd.github+json" --method GET "/repos/jamulussoftware/jamulus/pulls/0"
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/reference/pulls#get-a-pull-request"
}
gh: Not Found (HTTP 404)
```

Conclusion: `gh` properly logs errors, even with `--silent`. I don't see any benefit to have the request body in addition, nor do I see a way to easily do that (the body is always on stdout).

Therefore, I think it makes most sense to keep using `--silent` without further adjustments.
@pljones What do you think?

Reply:
OK, that's good enough.